helm/vertical-pod-autoscaler-app/values.yaml

```diff
@@ -84,24 +84,21 @@ vertical-pod-autoscaler:
 
     resources:
       limits:
-        cpu: 50m
-        memory: 250Mi
+        cpu: 100m
+        memory: 300Mi
       requests:
-        cpu: 50m
-        memory: 200Mi
+        cpu: 100m
+        memory: 250Mi
```

Comment:
Do we have a basis for those values?

Reply:
That's fine for me :) I'm wondering if we could get the values from existing clusters  in grafna cloud but that's alright for me